src/overlay/OverlayManagerImpl.cpp

```diff
@@ -201,10 +209,33 @@ OverlayManagerImpl::PeersList::acceptAuthenticatedPeer(Peer::pointer peer)
 
     CLOG(INFO, "Overlay") << "Non preferred " << mDirectionString
                           << " authenticated peer " << peer->toString()
-                          << " rejected because all available slots are taken.";
+                          << " rejected because all available slots are taken."
```

Comment:
I think these lines are out of order. Adding a function for this also would have avoided this issue.

Reply:
same thing: I do want this "@..." thing at the end of the line